http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HTTPAppDescriptorDAO.scala

```diff
@@ -0,0 +1,78 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package dao
+
+import cats.effect.Effect
+import cats.implicits._
+import cats.mtl.Ask
+import io.chrisdavenport.log4cats.StructuredLogger
+import io.circe.Decoder
+import io.circe.yaml.parser
+import org.broadinstitute.dsde.workbench.model.TraceId
+import org.broadinstitute.dsde.workbench.leonardo.JsonCodec._
+import org.broadinstitute.dsde.workbench.leonardo.dao.HttpAppDescriptorDAO._
+import org.http4s.{Method, Request, Response, Uri}
+import org.http4s.client.Client
+
+class HttpAppDescriptorDAO[F[_]](httpClient: Client[F])(implicit logger: StructuredLogger[F], F: Effect[F])
+    extends AppDescriptorDAO[F] {
+  override def getDescriptor(path: String)(implicit ev: Ask[F, TraceId]): F[AppDescriptor] =
+    for {
+      _ <- logger.info(s"Attempting to retrieve descriptor $path")
```

Comment:
I think some (all?) of these logs can be removed

Reply:
Yeah... They probably can, but its very helpful as circe can sometimes eat logs if somewhere along the line decoding fails. There are tests to verify it, but it could be annoying for future develops when adding new fields that aren't necessarily in the hard-coded yaml string in the test